blog/wort_zum_abschied/index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Ein Wort zum Abschied
```

Comment:
Also bei der Titelwahl hätte ich mir wirklich gewünscht, dass du dir mehr Gedanken machst und nicht einfach blind die Titel aus dem Buch übernimmst (ich nehme an, das ist, was hier passiert ist). Blogartikel, auch wenn sie Teil einer Reihe sind, müssten immer auch für sich alleine stehen können und ein Titel muss aussagekräftig genug sein, dass der Leser zumindest eine Idee hat, worum es geht ohne den Kontext zu kennen.

Stell dir vor du besuchst die Seite zum ersten mal und klickst auf "Blog", und dann ist der erste Artikel in der Liste "Ein Wort zum Abschied". Was glaubst du ist der erste Gedanke?

<img width="1287" alt="Screenshot 2023-09-14 at 09 08 44" src="https://github.com/pentacode/site/assets/1270940/e8fefc0a-54fc-441b-828c-eaa338fca0a7">